### ISSUE TITLE
UI improvement: Accessibility/Contrast Receive screen

### ISSUE THREAD

**Issue by mouxdesign** [ACINQ/phoenix#284]
![IMG_2025](https://user-images.githubusercontent.com/90313394/171156066-7c557817-c148-4519-9888-da7ebaad2d32.PNG)

Possible UI improvement: 
Text field boxes for 1) Amount 2) Description the color contrast ratios are quite low.  Increasing the field box to background ratio to 3:1 would also follow good accessibility principles. 

[Figma plugin](https://www.figma.com/community/plugin/748533339900865323/) to check contrast ratios.

**Comment by robbiehanson** [comment]
Thanks for reporting this:

Before:
<img width="248" alt="dark - before" src="https://user-images.githubusercontent.com/304604/171493752-a6c3b6f6-39e1-4abe-9e51-7a340b650b33.png">

After:
<img width="250" alt="dark - after" src="https://user-images.githubusercontent.com/304604/171493779-40830a09-9d30-4b8c-ac6e-ea9dc5905329.png">

Previously we were using `UIColor.separatorColor`, which is rgba(84,84,88,0.6) in dark mode. On a black background this gives a contrast ratio of 1.65.

For dark mode, we switched to `Color.systemGray2`, which is rgba(99,99,102,1.0), giving a contrast ratio of 3.5.

**Comment by mouxdesign** [comment]
Happy to help! That looks great. We were doing some accessibility testing on LN wallets and Phoenix was one that I tested myself and looked at the userflows. Will for sure contribute more in the future as I do more testing.
